docs/libcurl/curl_easy_setopt.md

```diff
@@ -471,6 +471,10 @@ See CURLOPT_HTTPPOST(3)
 
 Tunnel through the HTTP proxy. CURLOPT_HTTPPROXYTUNNEL(3)
 
+## CURLOPT_HTTPPROXYUDPTUNNEL
+
+UDP Tunnel through the HTTP proxy. CURLOPT_HTTPPROXYUDPTUNNEL(3)
```

Comment:
Can we elaborate on this? When or why would a user want to enable this and tunnel UDP explicitly?